sycl/unittests/device_code_split/src/device-code-first-file.cpp

```diff
@@ -0,0 +1,23 @@
+//==--- device-code-second-file.cpp --- Device code split unit tests -------==//
```

Comment:
I think, you don't need a subfolder for sources.

Reply:
**No**, I need a subfolder here, because inside the **add_sycl_unittest** function there is a check for the presence of unused sources, but presented in the current directory.

I checked without a subfolder and  faced a problem for each source file when using the **add_sycl_unittest** function, here is an example for **device-code-first-file.cpp**:
```
CMake Error at llvm/cmake/modules/LLVMProcessSources.cmake:114
Found unknown source file device-code-first-file.cpp
```